eng/common/build.ps1

```diff
@@ -110,6 +110,13 @@ function GetDotNetInstallScript([string] $dotnetRoot) {
 function InstallDotNetSdk([string] $dotnetRoot, [string] $version) {
   $installScript = GetDotNetInstallScript $dotnetRoot
   
+  if ($architecture -eq "arm") {
```

Comment:
Should we really be modifying common files?

Reply:
Also, if we move to helix, this will be interesting.We'll have an x86/x64 CLI to run tests with...

Reply:
For now, I modified the common script. It is necessary now because it would try to install the arm dotnet on the Windows cross (x86) build.